HCP Terraform Docs for break glass accounts (#2487)
hashicorp/web-unified-docs: content/terraform-docs-common/docs/cloud-docs/users-teams-organizations/single-sign-on/index.mdx

If an organization's owners disable SSO, all members can continue to access the organization using their HCP Terraform or HashiCorp Cloud Platform credentials.

### Break-glass Accounts

> Before you configure SSO for your organization, we recommend that you create and designate two "Break-glass" accounts with Owner permissions for your platform administrators. These accounts can still access the organization with their username and password, to allow you to troubleshoot any problems with your SSO provider.

Reviewer: "Break glass" is a colloquialism, but I know you're adding it because of SigXP.

Two options I see here:
1. Write this in terms of "Emergency access accounts", with "break glass" being mentioned (e.g., "These accounts are often referred to as **break glass accounts**")
2. Keep it in terms of break glass, but explain what it means (e.g., "Break glass accounts are used for emergency access." - not sure if getting into the "in case of emergency, break glass" etymology is needed)

I _think_ I have a slight preference for #1, but it's very very slight.

Author: So the pro-"break-glass account" argument is that it's a common term in the SSO provider space, eg: [here](https://support.auth0.com/center/s/article/Have-a-username-and-password-non-SSO-break-glass-account-with-the-same-email-address-for-SSO) and [here](https://entro.security/glossary/break-glass-account/).

Reviewer: That works for me!  Let's add a brief explanation (e.g., "also known as emergency access accounts" or "used for emergency access").

That at least makes the term "emergency" searchable for our 2AM sleep-deprived operator.

> Before you configure SSO for your organization, we recommend that you create and designate two "Break-glass" accounts with Owner permissions for your platform administrators. These accounts can still access the organization with their username and password, to allow you to troubleshoot any problems with your SSO provider.

Reviewer: Do we advise that the break glass accounts are not tied to a human, and aren't intended to be used for normal use, just troubleshooting?

Author: Clarified this.

> Before you configure SSO for your organization, we recommend that you create and designate two "Break-glass" accounts with Owner permissions for your platform administrators. These accounts can still access the organization with their username and password, to allow you to troubleshoot any problems with your SSO provider.

Reviewer: ```suggestion
Before you configure SSO for your organization, we recommend that you create and designate two "Break-glass" accounts with Owner permissions for your platform administrators. If you encounter problems with your SSO provider, you can use these accounts to access the organization for troubleshooting using their username and password.
```

> 

Reviewer 2: ```suggestion
### Emergency access accounts

Before you configure SSO for your organization, we recommend creating and designating two accounts with [owner permissions](/terraform/cloud-docs/users-teams-organizations/permissions/organization#organization-owners) that your platform administrators can use in case of an emergency. If you encounter problems with your SSO provider, you can use these accounts to access the organization for troubleshooting using their username and password.

```
I get wanting to use "break glass", but it is a [figure of speech](https://github.com/hashicorp/web-unified-docs/blob/main/docs/style-guide/general/language.md#do-not-use-figures-of-speech). It's clearer just to say that they are accounts that you can use for emergencies.

> Before you configure SSO for your organization, we recommend creating and designating two accounts with [owner permissions](/terraform/cloud-docs/users-teams-organizations/permissions/organization#organization-owners) that your platform administrators can use in case of an emergency. If you encounter problems with your SSO provider, you can use these accounts to access the organization for troubleshooting using their username and password.

Author: ```suggestion
Before you configure SSO for your organization, we recommend creating and designating two accounts with [owner permissions](/terraform/cloud-docs/users-teams-organizations/permissions/organization#organization-owners), sometimes referred to as "Break glass accounts". If you encounter problems with your SSO provider that prevent you from logging in with SSO credentials, use these accounts to access the organization for troubleshooting using their username and password.
```

> Before you configure SSO for your organization, we recommend creating and designating two accounts with [owner permissions](/terraform/cloud-docs/users-teams-organizations/permissions/organization#organization-owners), sometimes referred to as "Break glass accounts". If you encounter problems with your SSO provider that prevent you from logging in with SSO credentials, use these accounts to access the organization for troubleshooting using their username and password.

Reviewer: ```suggestion
Before you configure SSO for your organization, we recommend creating and designating two accounts with [owner permissions](/terraform/cloud-docs/users-teams-organizations/permissions/organization#organization-owners), sometimes referred to as **break glass accounts**. If you encounter problems with your SSO provider that prevent you from logging in with SSO credentials, use these accounts to access the organization for troubleshooting using their username and password.
```
